docs/x509.rst

```diff
@@ -718,6 +718,68 @@ X.509 Extensions
 
         :returns: A list of values extracted from the matched general names.
 
+.. class:: CertificatePolicies
+
+    .. versionadded:: 0.9
+
+    The certificate policies extension is a list of one or more
+    :class:`PolicyInformation` instances. The object is iterable to get every
+    instance.
+
+Certificate Policies Classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These classes may be present within a :class:`CertificatePolicies` instance.
+
+.. class:: PolicyInformation
+
+    .. versionadded:: 0.9
+
+    Contains a policy identifier and an optional list of qualifiers.
+
+    .. attribute:: policy_identifier
+
+        :type: :class:`ObjectIdentifier`
+
+    .. attribute:: policy_qualifiers
+
+        :type: list
+
+        A list of :class:`PolicyQualifierInfo` objects.
+
+.. class:: PolicyQualifierInfo
+
+    .. versionadded:: 0.9
+
+    .. attribute:: qualifier
+
+        :type: :term:`text` or :class:`UserNotice`
```

Comment:
I was looking at this more this morning and right now you actually can determine the type of `qualifier` by looking at the `policy_qualifier_id` property that is auto-calculated based on the type you pass in when creating the `PolicyQualifierInfo`. So the API for using this is:

```python
if pqi.policy_qualifier_id == OID_CPS_USER_NOTICE:
    # do a thing
else:
    # do the other thing
```